ngraph/test/threading.cpp

```diff
@@ -73,6 +73,8 @@ TEST(threading, check_atomic_guard) {
         result += 3;
     };
     const auto& thread2_fun = [&]() {
+        std::chrono::milliseconds ms{500};
```

Comment:
nit: there is still potential possibility that 'thread1' will start more than 0.5s later than thread2, but I guess it is ok for tests. Better way would be
```
std::atomic_bool thread1_started {false};
thread1: { AtomicGuard(flag); thread1_started = true; ... }
thread2: { while (!thread1_started); AtomicGuard(flag); ...}
```